public/assets/js/impact_discover/impact_discover.js

```diff
@@ -0,0 +1,261 @@
+/*
+@licstart  The following is the entire license notice for the
+JavaScript code in this page.
+
+Copyright (C) 2010  Goteo Foundation
+
+The JavaScript code in this page is free software: you can
+redistribute it and/or modify it under the terms of the GNU
+General Public License (GNU GPL) as published by the Free Software
+Foundation, either version 3 of the License, or (at your option)
+any later version.  The code is distributed WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS
+FOR A PARTICULAR PURPOSE.  See the GNU GPL for more details.
+
+As additional permission under GNU GPL version 3 section 7, you
+may distribute non-source (e.g., minimized or compacted) forms of
+that code without the copy of the GNU GPL normally required by
+section 4, provided you include this license notice and a URL
+through which recipients can access the Corresponding Source.
+
+
+@licend  The above is the entire license notice
+for the JavaScript code in this page.
+*/
+
+//reset ODS JSON
+function resetODS() {
+    $.each(odsList.ods, function(key, value){
+        odsList.ods[key]["active"] = false
+    })
+}
+
+// reset ODS select
+function resetODSSelect() {
+    $(".impact-discover-filters select option").remove();
+    $(".impact-discover-filters select").append("<option>Filtra por Objetivos de Desarrollo sostenible</option");
+}
+
+// check if ODS option has selected footprint
+function hasFootprint (ods, footprint) {
+    var footprints = ods.footprints;
+    return footprints.indexOf(footprint)>=0 ? true : false;
```

Comment:
@davidbeig Wouldn't this be the same than:
`return footprints.indexOf(footprint) >= 0;`